AirCasting/SessionViews/MultiSliderView.swift

```diff
@@ -6,19 +6,24 @@
 //
 
 import SwiftUI
+import Resolver
 
 struct ThresholdsSliderView: View {
     
     @ObservedObject var threshold: SensorThreshold
+    @Binding var selectedStream: MeasurementStreamEntity?
```

Comment:
I think you can remove this `@Binding`. The purpose of bindings is to allow to change a State variable of some by the child view. In this case, the slider doesn't change the value of `selectedStream`. I checked, and everything seems to be working fine without it.